services/streammanager/kafka/kafkamanager.go

```diff
@@ -130,27 +130,31 @@ func NewProducer(destinationConfig interface{}, o Opts) (sarama.SyncProducer, er
 
 	var destConfig = Config{}
 	jsonConfig, err := json.Marshal(destinationConfig)
+	hosts := make([]string, 0)
```

Comment:
[minor]
It would be preferable not to interrupt error checking with other code. 
```suggestion
```